packages/app/src/utils/eventToClip.ts

```diff
@@ -8,6 +8,8 @@ export function eventToClip(
   if (!(target_ instanceof HTMLElement)) {
     return vec2f()
   }
+  // TODO: cache getBoundingClientRect
```

Comment:
Would it be a good plan to open a gh issue for every todo, and perhaps add a ref to it #xx in code?